doop.c

```diff
@@ -635,6 +635,37 @@ Perl_do_trans(pTHX_ SV *sv)
     }
 }
 
+static int
+S_sv_has_magic_method(pTHX_ SV *sv, int method)
+{
+    HV* stash = NULL;
```

Comment:
that should read
```
HV *stash = NULL
```
for consistency and correctness.

Reply:
Also this should be using STATIC and it should be registered in embed.fnc and using the generated macros.